### PR TITLE
Bump xblock-eoc-journal version

### PR DESCRIPTION
**Pull request by snowcrshd**
Bump version to reflect latest changes to problem builder

**Reviewers**

[ ] @mtyaka

### CODE REVIEW

File: requirements/edx/custom.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@
 -e git+https://github.com/edx-solutions/xblock-adventure.git@7bdeb62b1055377dc04a7b503f7eea8264f5847b#egg=xblock-adventure
 -e git+https://github.com/open-craft/xblock-poll.git@copy-change-patch#egg=xblock-poll
 -e git+https://github.com/edx/edx-notifications.git@0.6.0#egg=edx-notifications==0.6.0
--e git+https://github.com/open-craft/problem-builder.git@d312e9a587e4376b9afadb982f5fc28e4a9061fc#egg=problem-builder
+-e git+https://github.com/open-craft/problem-builder.git@f2749cf50adf1ed053d71d907357e1d3e76e9e37#egg=problem-builder
```
Comment (mtyaka) [inline requirements/edx/custom.txt]: @snowcrshd Wrong xblock! :) Should be `xblock-eoc-journal` three lines below.

Comment (snowcrshd) [inline requirements/edx/custom.txt]: wtf, I don't know why I had pb in mind.

**Comment by snowcrshd** [comment]
@mtyaka done

**Comment by mtyaka** [comment]
:+1:

- [x] I tested this: Installed the new version and verified it works.
- [x] I read through the code
- [x] I checked for accessibility issues
- [ ] ~~Includes documentation~~ not required

**Pull request merged**